esti/main_test.go

```diff
@@ -269,7 +269,7 @@ func deleteAllPolicies(ctx context.Context, client api.ClientWithResponsesInterf
 
 func TestMain(m *testing.M) {
 	systemTests := flag.Bool("system-tests", false, "Run system tests")
-	useLocalCredentials := flag.Bool("use-local-credentials", true, "Generate local API key during `lakefs setup'")
+	useLocalCredentials := flag.Bool("use-local-credentials", false, "Generate local API key during `lakefs setup'")
```

Comment:
This can break other tests, for instance I believe some Cloud tests run Esti.
But that's a consequence of an underlying issue: what changed? Why do we need to change this? (Can we perhaps just add the flag to where we run Esti?)